src/shared/components/Button.tsx

```diff
@@ -5,13 +5,25 @@ import React from 'react';
 interface ButtonProps {
   children: React.ReactNode;
   className?: string;
-  variant?: 'outline';
+  variant?: 'default' | 'outline'; // Added default variant
 }
 
-export function Button({ children, className }: ButtonProps) {
+export function Button({
+  children,
+  className = '',
+  variant = 'default',
+}: ButtonProps) {
+  const baseClasses =
```

Comment:
doesn't seem like this should be inside the function component, would re-define on every render.